### PR TITLE
[Feature] Add job application notes field & "add todo" dropdown exclude term

### LINKED ISSUE #420: [Feature] Add job application notes field & "add todo" dropdown exclude term employees.
job application notes field in the application and in the applicant 
action items "add todo" dropdown exclude term employees.

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #420

### CODE REVIEW

**Comment by James1x0** [comment]
Merge with associatedemployers/igneous#152

File: app/components/modals/add-notes.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+  modalId: computed('elementId', function () {
+    return this.get('elementId') + '-modal';
+  }),
+
+  createConfirm () {
+    $('#' + this.get('modalId')).modal({
+      detachable: true,
+      closable:   false,
+      context:    'body.ember-application'
```
Comment (James1x0) [inline app/components/modals/add-notes.js]:
```suggestion
      context:    '.ember-application'
```

File: app/templates/account/job-opening/campaign/job-application/index.hbs [hunk 2]
```diff
@@ -19,15 +19,66 @@
...
+    }}
+      {{#ui-popup position="bottom left" content="add notes"}}
+        <div class="content clearfix">
+          <span class="left floated">
+            <a href="#" {{action openAddModal}}>
+              <i class="edit icon"></i>
+            </a>
+          </span>
+          <span>
+            <h3 class="header">Notes</h3>
```
Comment (James1x0) [inline app/templates/account/job-opening/campaign/job-application/index.hbs]: I think the header class is pointless here.

File: app/templates/account/job-opening/campaign/job-application/index.hbs [hunk 3]
```diff
@@ -19,15 +19,66 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (James1x0) [inline app/templates/account/job-opening/campaign/job-application/index.hbs]: Don't leave dead code, we have to clean it up regardless.

File: app/templates/components/modals/add-notes.hbs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+        name="note-text"
+        value=model.notes
+        spellcheck="true"
+        rows=6
+        placeholder="Add notes"}}
+    </form>
+  </div>
+  <div class="actions">
+    <button class="ui inverted red button" {{action 'cancel'}}>
+      Cancel
```
Comment (James1x0) [inline app/templates/components/modals/add-notes.hbs]:
```suggestion
      Discard Changes
```

File: tests/integration/components/modals/add-notes-test.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+    // Template block usage:
+    await render(hbs`
+      {{#modals/add-notes}}
+        template block text
+      {{/modals/add-notes}}
+    `);
+
+    assert.equal(this.element.textContent.trim(), 'template block text');
+  });
+});
```
Comment (James1x0) [inline tests/integration/components/modals/add-notes-test.js]: Axe this file.

File: app/templates/components/modals/add-notes.hbs (hunk 4 above)
Comment (JavaLavaMT) [reply app/templates/components/modals/add-notes.hbs]: ✔️

File: app/templates/account/job-opening/campaign/job-application/index.hbs (hunk 3 above)
Comment (JavaLavaMT) [reply app/templates/account/job-opening/campaign/job-application/index.hbs]: ✔️

File: app/templates/account/job-opening/campaign/job-application/index.hbs (hunk 2 above)
Comment (JavaLavaMT) [reply app/templates/account/job-opening/campaign/job-application/index.hbs]: ✔️

File: app/components/modals/add-notes.js (hunk 1 above)
Comment (JavaLavaMT) [reply app/components/modals/add-notes.js]: 🏁

File: tests/integration/components/modals/add-notes-test.js (hunk 5 above)
Comment (JavaLavaMT) [reply tests/integration/components/modals/add-notes-test.js]: 🏁

**Pull request merged**